app/templates/play/ladder_home.jade

```diff
@@ -23,5 +23,8 @@ block content
               if playCount
                 span.spl.spr - #{playCount.sessions}
                 span(data-i18n="play.players") players
+                if view.levelStatusMap[level.id]
+                  | &nbsp;-&nbsp;
+                  span(class="level-status-#{view.levelStatusMap[level.id]}", data-i18n="play.level_status_#{view.levelStatusMap[level.id]}") #{view.levelStatusMap[level.id]}
```

Comment:
Looks like we already have "Complete" and "Started" translated for the clans page. This element should use `clans.complete_2` and `clans.started_2` for its `data-i18n` value, and should not create those values with string interpolation, so that in the future it will be clear that those strings are being used here with a text search. Also, the element should have no text in it, so that it's clear if it's properly localized.